examples/runners/ash/src/main.rs

```diff
@@ -197,6 +197,15 @@ pub fn main() {
 }
 
 pub fn compile_shaders() -> Vec<SpvFile> {
+    // Hack: spirv_builder builds into a custom directory if running under cargo, to not
```

Comment:
Since this is in example code, maybe make a note that this is not required for normal use and is there to make developing rust-gpu itself easier.

Reply:
This is not my comment, I just replicated it from the `wgpu` runner:
https://github.com/EmbarkStudios/rust-gpu/blob/6dc23afaf80f5917f11ccc8ea66f59b10aef98d4/examples/runners/wgpu/src/lib.rs#L96-L103

(one hint is `// Hack:`, I always write `// HACK:` or `// HACK(eddyb):`, heh)